src/erlcloud_sns.erl

```diff
@@ -258,10 +260,10 @@ set_endpoint_attributes(EndpointArn, Attributes, Config) ->
 
 
 
--spec list_endpoints_by_platform_application/1 :: (string()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
--spec list_endpoints_by_platform_application/2 :: (string(), undefined|string()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
--spec list_endpoints_by_platform_application/3 :: (string(), undefined|string(), aws_config()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
--spec list_endpoints_by_platform_application/4 :: (string(), undefined|string(), string(), string()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
+-spec list_endpoints_by_platform_application/1 :: (parameter()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
+-spec list_endpoints_by_platform_application/2 :: (parameter(), undefined|parameter()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
+-spec list_endpoints_by_platform_application/3 :: (parameter(), undefined|parameter(), aws_config()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
+-spec list_endpoints_by_platform_application/4 :: (parameter(), undefined|parameter(), parameter(), parameter()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}].
```

Comment:
Line 266 is too long: -spec list_endpoints_by_platform_application/4 :: (parameter(), undefined|parameter(), parameter(), parameter()) -> [{endpoints, [sns_endpoint()]} | {next_token, string()}]..